CrowServe/CrowServe.h

```diff
@@ -2,12 +2,35 @@
 
 #include "../shared_common.h"
 
-#include <sys/types.h>
-#include <sys/socket.h>
-#include <netdb.h>
 #include <unistd.h>
 #include <iostream>
-#include <string.h>
+
+#define PlatformWindows  1
+#define PlatformMac      2
+#define PlatformUnix     3
+
+#if defined(_WIN32)
+#define Platform PlatformWindows
+#elif defined(__APPLE__)
+#define Platform PLATFORM_MAC
+#else
+#define Platform PlatformUnix
+#endif
```

Comment:
This is not how this should be done in cpp. If you want alias you would do #define Windows, and then do #ifdef Windows, rather than having to do comparisons, that's very non standard